services/pane.js

```diff
@@ -215,56 +225,59 @@ function addPreview(preview) {
 
 /**
  * format and assemble error messages
- * @param {Object[]} errors
+ * @param {Object[]} errors (or warnings)
+ * @param {string} [modifier] modifier class for warnings, info, etc
  * @returns {Element}
  */
-function addErrors(errors) {
+function addErrors(errors, modifier) {
```

Comment:
Like what's going on with all of this. Very clear, except would like to have this function renamed something like `addErrorsOrWarnings()` to keep that clarity.